### PR TITLE
Change hljs theme & update build.cmd

### PR DESCRIPTION
**Pull request by qinezh**
@hellosnow @ansyral @superyyrrzz @vwxyzh @chenkennt

### CODE REVIEW

**Comment by qinezh** [comment]
#540

File: UpdateTemplate.cmd [hunk 1]
```diff
@@ -23,8 +24,16 @@ IF NOT '%ERRORLEVEL%'=='0' (
     GOTO :Exit
 )
 
+:: Check if bower install successfully
```
Comment (hellosnow) [inline UpdateTemplate.cmd]: >install [](start = 18, length = 7)

is installed

File: .gitignore [hunk 2]
```diff
@@ -45,4 +45,7 @@ project.lock.json
 ###############
 #  template   #
 ###############
-src/docfx/Template/*.zip
\ No newline at end of file
+src/docfx/Template/*.zip
+src/docfx.website.themes/default/styles/docfx.vendor.css
+src/docfx.website.themes/default/styles/docfx.vendor.js
+src/docfx.website.themes/default/fonts/
```
Comment (hellosnow) [inline .gitignore]: >fonts [](start = 33, length = 5)

is it deleted in this review? shall we still ignore?

Comment (qinezh) [inline .gitignore]: fonts will be copied from the bootstrap folder while updating template

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by vwxyzh** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**